rpc/rpc-lib/src/rpcsvc.c

```diff
@@ -375,12 +376,12 @@ rpcsvc_program_notify(rpcsvc_listener_t *listener, rpcsvc_event_t event,
     list_for_each_entry(wrapper, &listener->svc->notify, list)
     {
         if (wrapper->notify) {
-            wrapper->notify(listener->svc, wrapper->data, event, data);
+            ret = wrapper->notify(listener->svc, wrapper->data, event, data);
```

Comment:
This way we'll only return the error from the last notify, which could be a success even if a previous one failed. Probably we should take the first error < 0 as the return value

Reply:
@xhernandez Yes, I forgot that. I will fix it